sets/spike/prime/hand-controlled-grabber/README.md

```diff
@@ -0,0 +1,31 @@
+---
+permalink: /projects/sets/spike/prime/hand-controlled-grabber/
+title: "LEGO® Education SPIKE™ Prime: Hand-Controlled Grabber"
+maintainer:
+    user: "TheVinhLuong102"
+    name: "The Lương-Phạm Family"
+image:
+    local: "hand-controlled-grabber.png"
+    credit: "LEGO"
+video:
+    youtube: "https://www.youtube.com/embed/8uyx5npo4LA"
+description:
+    "Use the Grabber to pick up objects and bring them around!"
+building_instructions:
+    external: https://education.lego.com/v3/assets/blt293eea581807678a/blt56a81c75560c9a81/5f8802cbf71916144453a493/supercleaup-bi-pdf-book1of3.pdf
+---
+
+
+# LEGO® Education SPIKE™ Prime: Hand-Controlled Grabber
```

Comment:
This title is redundant now since we already have `title: "LEGO® Education SPIKE™ Prime: Hand-Controlled Grabber"` above.

```suggestion
```